### PR TITLE
Fix dependency issue

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Don't depend on bot code in App Engine.

### CODE REVIEW

File: src/clusterfuzz/_internal/base/task_utils.py [hunk 1]
```diff
@@ -12,7 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Module providing utilities for utask users. This module should not depend on
-any other module in tasks to prevent circular imports."""
+any other module in tasks to prevent circular imports and issues with
```
Comment (letitz) [inline src/clusterfuzz/_internal/base/task_utils.py:15]: Now that it's moved to `base/`, do we still need this warning? It seems to me that the directory structure implies this already.

**Review by letitz — APPROVED** [review]
LGTM with a single optional comment.

File: src/clusterfuzz/_internal/base/task_utils.py (hunk 1 above)
Comment (jonathanmetzman) [reply src/clusterfuzz/_internal/base/task_utils.py:15]: I don't think it's implied.

**Comment by jonathanmetzman** [comment]
This faces the appengine 500 error we are seeing.

**Pull request merged**